listenbrainz_spark/recommendations/create_dataframes.py

```diff
@@ -17,6 +19,8 @@
 
 # dataframe html is generated when set to true
 SAVE_DATAFRAME_HTML = False
+# os.path.join ignores everything before a '/'
+HDFS_PATH = config.HDFS_CLUSTER_URI + path.DATAFRAME_DIR + '/'
```

Comment:
@mayhem  I really need your comments on this.

Reply:
Yes, os.path.join is to join filenames consistently across various filesystems (windows, linux, mac). However, given that we're joining paths for HDFS and that this isn't really supported in os.path, I'm ok with this.